Source/WebCore/platform/graphics/mac/FontCustomPlatformDataMac.cpp

```diff
@@ -101,4 +102,20 @@ bool FontCustomPlatformData::supportsFormat(const String& format)
         || equalLettersIgnoringASCIICase(format, "woff"_s);
 }
 
+bool FontCustomPlatformData::supportsTech(const FontTech& tech)
```

Comment:
`FontCustomPlatformDataMac.cpp` doesn't seem right, since this isn't Mac-specific. Can we create a new file at `platform/graphics/coretext/FontCustomPlatformDataCoreText.cpp` instead?

Reply:
(Ideally this stuff would be in PAL, but that's probably out of scope for this patch...)